Documentation/nfs.md

```diff
@@ -80,15 +81,20 @@ metadata:
   name: rook-nfs
   namespace: rook-nfs
 spec:
+  serviceAccountName: rook-nfs
   replicas: 1
   exports:
-  - name: nfs-share
+  - name: share1
+    createStorageClass: true
```

Comment:
can these instances of `createStorageClass` be removed now from the examples?

Reply:
done